blog/models.py

```diff
@@ -38,7 +39,7 @@ class CLIP(models.Model):
     description = models.TextField(max_length=200, blank=True)
     barcode_file = models.CharField(max_length=25, choices=bar_choices, blank=True)
     chrom_sizes = models.CharField(max_length=50, choices=chrom_choices, blank=True)
-    star_index = models.CharField(max_length=50, choices=star_choices, blank=True)
+    star_index = MultiSelectField(choices=star_choices)
```

Comment:
I'm not able to get this working quite right, have you hit this error? Maybe we can use CharField for now, as I think there might be other ways to do multiselect without needing an older package (multiselect-field might not be compatible or updated to the latest Django)
```
Traceback (most recent call last):
  File "/Users/brianyee/PycharmProjects/metadata-db/manage.py", line 22, in <module>
    main()
  File "/Users/brianyee/PycharmProjects/metadata-db/manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "/Users/brianyee/PycharmProjects/metadata-db-env/lib/python3.9/site-packages/django/core/management/__init__.py", line 446, in execute_from_command_line
    utility.execute()
  File "/Users/brianyee/PycharmProjects/metadata-db-env/lib/python3.9/site-packages/django/core/management/__init__.py", line 420, in execute
    django.setup()
  File "/Users/brianyee/PycharmProjects/metadata-db-env/lib/python3.9/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/brianyee/PycharmProjects/metadata-db-env/lib/python3.9/site-packages/django/apps/registry.py", line 116, in populate
    app_config.import_models()
  File "/Users/brianyee/PycharmProjects/metadata-db-env/lib/python3.9/site-packages/django/apps/config.py", line 269, in import_models
    self.models_module = import_module(models_module_name)
  File "/Users/brianyee/miniconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/brianyee/PycharmProjects/metadata-db/blog/models.py", line 25, in <module>
    class CLIP(models.Model):
  File "/Users/brianyee/PycharmProjects/metadata-db/blog/models.py", line 42, in CLIP
    star_index = MultiSelectField(choices=star_choices)
  File "/Users/brianyee/PycharmProjects/metadata-db-env/lib/python3.9/site-packages/multiselectfield/db/fields.py", line 72, in __init__
    self.validators[0] = MaxValueMultiFieldValidator(self.max_length)
IndexError: list assignment index out of range
(metadata-db-env) (base) brians-mbp:metadata-db brianyee$ 

```

Reply:
that makes sense, it worked with Django 3.2.14 but after updating to Django 4.1.1 it seems to introduce this same error. I'll convert it back to charfield and find other ways to do this later